add details running on Windows (#690)
apache/dolphinscheduler-website: README.md

1. Run `npm install` in the root directory to install the dependencies.
2. Run `npm run start` in the root directory to start a local server, you will see the website in 'http://localhost:8080'.
```

> Note:if you clone the code in Windows not Mac .please read details as below.

Reviewer: ```suggestion
Note: if you clone the code in Windows, not Mac. Please read the details below.
```

Reviewer: Note the space separators.

> if you execute the commands like two steps,you will get exception "UnhandledPromiseRejectionWarning: Error: EPERM: operation not permitted, symlink '2.0.3' -> 'latest'".

Reviewer: ```suggestion
If you execute the commands like the two steps above, you will get the exception "UnhandledPromiseRejectionWarning: Error: EPERM: operation not permitted, symlink '2.0.3' -> 'latest'".
```

Reviewer: Does this express it a little more accurately?

> When you meet this problem.you will run two steps in the cmd.exe as an  ADMINISTRATOR MEMBER.

Reviewer: ```suggestion
When meeting this problem. You can run two steps in the cmd.exe as an  ADMINISTRATOR MEMBER.
```

> When meeting this problem. You can run two steps in the cmd.exe as an  ADMINISTRATOR MEMBER.

Reviewer 2: ```suggestion
When meeting this problem. You can run two steps in the cmd.exe as an ADMINISTRATOR MEMBER.
```
